.github/workflows/pytests.yml

```diff
@@ -6,13 +6,51 @@ on:
       - main
   pull_request:
 
+
 jobs:
   test:
+    name: Run tests and collect coverage
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: gardenlinux/python-gardenlinux-lib/.github/actions/setup@main
-      - name: Run tests
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Setup Gardenlinux
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/setup@main
+
+      - name: Activate caching based on lockfile
```

Comment:
This step can be removed for `pytests.yml` as it will be part of #156.

Reply:
Ah yes, missed that one, thanks!